cli/src/commands/extensions/mod.rs

```diff
@@ -273,9 +274,16 @@ fn ask_permissions(extension: &Extension) -> Result<()> {
         }
     }
 
-    print_permissions_list("Read", "path", permissions.read.get());
+    // Special case `run = true` to always say "Read any path"
+    if let Permission::Boolean(true) = &permissions.run {
```

Comment:
I don't understand this special case? What's the goal here? Just to reduce the duplication between run and read? Seems weird if we don't have that consistently for everything.